spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+Encoding.default_external = Encoding::UTF_8
```

Comment:
you should probably put this in the code though:

https://github.com/chef/omnibus/blob/master/lib/omnibus.rb

Reply:
https://github.com/berkshelf/berkshelf/blob/master/lib/berkshelf.rb#L28-L29

Reply:
I'm open to considering that for our greater set of projects.